### PR TITLE
[refactor] Remove legacy use of myItems plugin. Create new myItems pl…

### PR DESCRIPTION
**Pull request by shefalijoshi**
…ugin using ObjectProvider

### CODE REVIEW

File: src/plugins/myItems/plugin.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
```
Comment (psarram) [inline src/plugins/myItems/plugin.js]: Copyright should be 2014-2019.

File: src/plugins/plugins.js [hunk 2]
```diff
@@ -69,11 +70,11 @@ define([
     Filters,
     ObjectMigration,
     GoToOriginalAction,
-    ClearData
+    ClearData,
+    MyItems
 ) {
     var bundleMap = {
         LocalStorage: 'platform/persistence/local',
-        MyItems: 'platform/features/my-items',
```
Comment (psarram) [inline src/plugins/plugins.js]: Please also delete this directory.

File: src/plugins/myItems/plugin.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            });
+
+            let objectProvider = {
+                get: function (identifier) {
+                    if (identifier.key === idKey) {
+                        return Promise.resolve({
+                            identifier: identifier,
+                            name: domainObjectName,
+                            type: domainObjectType,
+                            location: domainObjectLocation,
```
Comment (psarram) [inline src/plugins/myItems/plugin.js]: Perhaps type ('folder') can be inlined here? Same with location?

File: src/plugins/myItems/plugin.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+define([
+], function (
+) {
+    const idNamespace = '',
+        idKey = 'mine',
```
Comment (psarram) [inline src/plugins/myItems/plugin.js]: I would name these namespace and key for consistency.

**Comment by shefalijoshi** [comment]
Closing this PR (the fork branch) and creating a new one with the nasa/openmct repo.

**Pull request closed without merging**